articles/active-directory/enterprise-users/licensing-service-plan-reference.md

```diff
@@ -48,6 +48,7 @@ When [managing licenses in the Azure portal](https://portal.azure.com/#blade/Mic
 | Microsoft Entra ID P1_USGOV_GCCHIGH | AAD_PREMIUM_USGOV_GCCHIGH |	de597797-22fb-4d65-a9fe-b7dbe8893914 | EXCHANGE_S_FOUNDATION (113feb6c-3fe4-4440-bddc-54d774bf0318)<br/>MFA_PREMIUM (8a256a2b-b617-496d-b51b-e76466e88db0)<br/>ADALLOM_S_DISCOVERY (932ad362-64a8-4783-9106-97849a1a30b9)<br/>AAD_PREMIUM (41781fb2-bc02-4b7c-bd55-b576c07bb09d) | Exchange Foundation (113feb6c-3fe4-4440-bddc-54d774bf0318)<br/>Microsoft Azure Multi-Factor Authentication (8a256a2b-b617-496d-b51b-e76466e88db0)<br/>Microsoft Defender for Cloud Apps Discovery (932ad362-64a8-4783-9106-97849a1a30b9)<br/>Microsoft Entra ID P1 (41781fb2-bc02-4b7c-bd55-b576c07bb09d) |
 | Microsoft Entra ID P2	| AAD_PREMIUM_P2	| 84a661c4-e949-4bd2-a560-ed7766fcaf2b	| AAD_PREMIUM (41781fb2-bc02-4b7c-bd55-b576c07bb09d)<br/>AAD_PREMIUM_P2 (eec0eb4f-6444-4f95-aba0-50c24d67f998)<br/>ADALLOM_S_DISCOVERY (932ad362-64a8-4783-9106-97849a1a30b9)<br/>EXCHANGE_S_FOUNDATION (113feb6c-3fe4-4440-bddc-54d774bf0318)<br/>MFA_PREMIUM (8a256a2b-b617-496d-b51b-e76466e88db0) | Microsoft Entra ID P1 (41781fb2-bc02-4b7c-bd55-b576c07bb09d)<br/>Microsoft Entra ID P2 (eec0eb4f-6444-4f95-aba0-50c24d67f998)<br/>CLOUD APP SECURITY DISCOVERY (932ad362-64a8-4783-9106-97849a1a30b9)<br/>EXCHANGE FOUNDATION (113feb6c-3fe4-4440-bddc-54d774bf0318)<br/>MICROSOFT AZURE MULTI-FACTOR AUTHENTICATION (8a256a2b-b617-496d-b51b-e76466e88db0) |
 | Azure Information Protection Plan 1	| RIGHTSMANAGEMENT	| c52ea49f-fe5d-4e95-93ba-1de91d380f89	| RMS_S_ENTERPRISE (bea4c11e-220a-4e6d-8eb8-8ea15d019f90)<br/>RMS_S_PREMIUM (6c57d4b6-3b23-47a5-9bc9-69f17b4947b3) | AZURE INFORMATION PROTECTION PREMIUM P1 (6c57d4b6-3b23-47a5-9bc9-69f17b4947b3)<br/>Microsoft Entra RIGHTS (bea4c11e-220a-4e6d-8eb8-8ea15d019f90) |
+| Azure Information Protection Plan 1	| RIGHTSMANAGEMENT_CE	| a0e6a48f-b056-4037-af70-b9ac53504551	| RMS_S_ENTERPRISE (bea4c11e-220a-4e6d-8eb8-8ea15d019f90)<br/>RMS_S_PREMIUM (6c57d4b6-3b23-47a5-9bc9-69f17b4947b3) | EXCHANGE_S_FOUNDATION (113feb6c-3fe4-4440-bddc-54d774bf0318) |
```

Comment:
The Service Plans entries aren't quite right. Here is the correct markup: 

| Azure Information Protection Plan 1 |	RIGHTSMANAGEMENT_CE | a0e6a48f-b056-4037-af70-b9ac53504551 |	EXCHANGE_S_FOUNDATION (113feb6c-3fe4-4440-bddc-54d774bf0318)<br/>RMS_S_PREMIUM (6c57d4b6-3b23-47a5-9bc9-69f17b4947b3)<br/>RMS_S_ENTERPRISE (bea4c11e-220a-4e6d-8eb8-8ea15d019f90) | Exchange Foundation (113feb6c-3fe4-4440-bddc-54d774bf0318)<br/>Azure Information Protection Premium P1 (6c57d4b6-3b23-47a5-9bc9-69f17b4947b3)<br/>Azure Rights Management (bea4c11e-220a-4e6d-8eb8-8ea15d019f90) |

Reply:
Thank you for correcting them. This was my first attempt at digging into them and trying to submit a detailed addition, rather than just saying "This is missing ...." I poked around a little more and realized I was missing a few things.  I would like to learn more about maintaining this page.

Reply:
Thanks @mprentice22153 I really appreciate your help in leaning in here to help to make edits. Regarding this particular SKU you actually had all of the service plans in your PR but what was incorrect was two things. 
1. you had the service plan part numbers spread across two columns. The Exchange one should have been included in the same column as the two RMS service plans. 
2. you didn't include the service plan friendly names in the last column. This is essentially the same list as the service plans column but repeated with the friendly names instead of the part numbers. 

Thanks
Nick

Reply:
Because the friendly names don't display in the webpage output I missed adding those and didn't realize they were there, even though that is the main reason I download the CSV regularly into my scripts.

What is the correct way for me to re-edit/re-submit the correct entries?
By updating the markup does it then also update the CSV, or is that a different edit?

Reply:
You will either have to edit this PR and insert the additional information or close this PR and start a new one. And no this edit will not fix the CSV, I will have to do that myself. I will get that done shortly.